server/api/observation/observation.controller.create.ts

```diff
@@ -0,0 +1,27 @@
+import { getManager } from 'typeorm';
+import { Response } from 'express';
+import { ApiRequest, EdipiParam } from '../index';
+import ObservationApiI from './observation.api-i';
+import { timestampColumnTransformer } from '../../util/util';
+
+
+class ObservationControllerCreate {
```

Comment:
Getting and saving entities are two different concerns and they should not be mixed up in one module. There should be a single module for a single concern.

Reply:
In what way will this improve the codebase? I'd like to discuss it some more since it's a sweeping new convention you're proposing. These sorts of convention changes should be discussed before implementing to ensure that they bring value to the codebase, aren't just a stylistic preference, and consistency is maintained across the project.

This seems like a somewhat arbitrary distinction to me as far as code structure is concerned for this project, but I'm curious how you think this may help.

Reply:
fixed